src/Configuration.KeyPerFile/test/Microsoft.Extensions.Configuration.KeyPerFile.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>$(DefaultNetCoreTargetFramework);net472</TargetFrameworks>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework);$(DefaultNetFxTargetFramework)</TargetFrameworks>
```

Comment:
I'm not sure why we were previously targeting `net472` here and in a few other test projects. Please look at the blame and check w/ those responsible because we may need to test using _both_ `net461` and `net472`.

Reply:
These all come from dotnet/extensions, and have been net472 as far back as the history goes - they were originally set to net472 by Nate. We could support both, but I think it should be fine to just use net461 everywhere

Reply:
@dougbu any concerns about this? If not this is ready to merge

Reply:
No, I'm comfy